docs/developer-guide/developing.md

```diff
@@ -122,9 +122,11 @@ In order to have a full running MapStore in development environment, you need to
 This runs automatically with `npm start`. If you want to run only the backend, you can use `npm run be:start`.
 
 The back end will run on port 8080 and will look for the front-end at port 8081. If you want to change the back-end port, you can set the environment variable `MAPSTORE_BACKEND_PORT` to the desired port.
+Optionally you can set the data dir location by setting the environment variable `MAPSTORE_DATA_DIR`.
 
 ```sh
-export MAPSTORE_BACKEND_PORT=8082
+export MAPSTORE_BACKEND_PORT=8082 # set a different backend port
+export MAPSTORE_DATA_DIR=/usr/datadir # set the datadir location
```

Comment:
i'm on a windows machine and i tried opening a prompt from the root of the project and running the following

```
set MAPSTORE_BACKEND_PORT=8082
set MAPSTORE_DATA_DIR=C:\Users\user\Documents\datadir
npm run be:start
```
![image](https://github.com/geosolutions-it/MapStore2/assets/11991428/08946a5b-7fae-495d-9e08-997728eb9b8c)

but datadir is not recognized 
only the port is used

the frontend is not loading the correct localConfig.json placed into the datadir folder


then i tried and it did not worked anyway
`npm run be:start -- -Ddatadir.location=C:\Users\user\Documents\datadir`

Reply:
there was a missing part in the documentation that we integrated, basically we have also to edit the proxy